src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        // reading from file
+        StringBuilder builder = new StringBuilder();
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            int value = reader.read();
+            while (value != -1) {
+                builder.append((char)value);
+                value = reader.read();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file", e);
+        }
+
+        // changing StringBuilder into String array
+        String[] data = builder.toString().split("\n");
```

Comment:
Instead of manually reading the file character by character, consider using `reader.readLine()` which reads a line of text. This would simplify the code and make it more readable.